docs/catalog/ai/pre-trained-transfer-learning.md

```diff
@@ -0,0 +1,35 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: ai
+tags: 
+ - ai
```

Comment:
Should we add `machine-learning` as a tag here as well since it is being referenced in the pattern content?